node/node.go

```diff
@@ -1207,6 +1234,9 @@ func (app *App) initServices(ctx context.Context) error {
 		activation.WithPoets(poetClients...),
 		activation.BuilderAtxVersions(app.Config.AtxVersions),
 	)
+	if nodeServiceClient != nil {
+		atxBuilder.AlwaysSynced = true
```

Comment:
NIT: Instead you could pass something like this to the `NewBuilder()`:

```go
type alwaysSyncedSyncer struct{}

func (s alwaysSyncedSyncer) RegisterForATXSynced() <-chan struct{} {
	ch := make(chan struct{})
	close(ch)
	return ch
}
```

Reply:
We can do that later, when we will set app.syncer to some interface instead of a particular struct used at the moment.
This interface will need to provide also Start/Close methods and probably more as we are passing it as an arg to e.g. `NewTransactionService` and some other places (`NewNodeService` already is expecting an interface with mentioned method and also IsSynced method).

Reply:
done